src/main/resources/static/api.yml

```diff
@@ -0,0 +1,616 @@
+    description: Available commands for admin / auth
+paths:
+  /treatments:
+    get:
+      tags:
+        - Doctors
+      summary: Get all treatments
+      description: ''
+      operationId: getAllTreatments
+      responses:
+        '200':
+          description: All treatments successfully fetched
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Treatment'
+  /patients:
+    get:
+      tags:
+        - Doctors
+        - Nurses
+      summary: Get all patient files
+      description: ''
+      operationId: getAllPatients
+      responses:
+        '200':
+          description: Patient files successfully fetched
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Patient'
+    post:
+      tags:
+        - Doctors
+      summary: Add a patient
+      description: Adds provided patient to database
+      operationId: addPatient
+      requestBody:
+        description: Patient info to be added to the database
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/NewPatient'
+      responses:
+        '200':
+          description: Patient successfully added to database
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Patient'
+  /patients/{patientId}:
+    get:
+      tags:
+        - Doctors
+        - Nurses
+      summary: Get patient file with the given ID
+      description: ''
+      operationId: getPatient
+      parameters:
+        - name: patientId
+          in: path
+          description: Unique patient identifier from Patients table
+          required: true
+          schema:
+            type: string
+            example: UUID
+      responses:
+        '200':
+          description: Patient file successfully fetched
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Patient'
+    put:
+      tags:
+        - Doctors
+      summary: Edit patient file of the given ID
+      description: ''
+      operationId: editPatient
+      parameters:
+        - name: patientId
+          in: path
+          description: ID of the patient that will be edited
+          required: true
+          schema:
+            type: string
+            example: UUID
+      requestBody:
+        description: Patient's JSON file that will replace existing one
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/NewPatient'
+
+      responses:
+        '200':
+          description: patient successfully modified
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Patient'
+    delete:
```

Comment:
1) we don't delete patient here, but generally do edit action. so it should be PUT request
2) As soon as we even don't edit patient itself, let's make this endpoint `/patients/{patientId}/discharge`

Reply:
Ok, added those modifications to endpoint ` /patients/{patientId}/discharge` this commit.